howdoi/howdoi.py

```diff
@@ -578,12 +592,15 @@ def howdoi(raw_query):
     res = cache.get(cache_key)  # pylint: disable=assignment-from-none
 
     if res:
+        info_logging('Using cached response (add -C to clear the cache)')
         return _parse_cmd(args, res)
 
+    info_logging('Fetching answers for query: {}'.format(args['query']))
+
     try:
         res = _get_answers(args)
         if not res:
-            res = {'error': 'Sorry, couldn\'t find any help with that topic\n'}
+            res = {'error': 'Sorry, couldn\'t find any help with that topic\n(use --explain to learn why)'}
```

Comment:
We should check if the `--explain` flag is set before including this message.

Reply:
will be using logical AND. So this message will only be set if not res AND -x is false